src/components/tutorial/UnitProgressIcon.astro

```diff
@@ -122,6 +122,14 @@ const t = useTranslations(Astro);
 		background-color: var(--theme-bg-offset);
 	}
 
+	:global([aria-selected='true']) .segment:not(.done) {
+		stroke: hsl(var(--color-gray-20));
+	}
+
+	:global(.theme-dark [aria-selected='true']) .segment:not(.done) {
+		stroke: hsl(var(--color-gray-70));
+	}
+
```

Comment:
While it’s true that now you can see the empty segments better, doesn’t this now mean the contrast between complete/incomplete segments is too low?

If we remove colours it’s very subtle that these are different:

<img width="124" alt="image" src="https://user-images.githubusercontent.com/357379/225456408-fea4e118-5b19-4325-910c-443373155bcc.png">

I’d suggest making the empty segments only much more slightly lighter than they are currently (or leaving as-is — I think arguably it’s the complete segments that are the key information and you can see the empty space in contrast to those).

Reply:
Tweaked these a bit. I guess this works as a compromise, but I'm fine with leaving it as-is if you still think it isn't enough contrast. I reduced the saturation of the images to zero to try emulate what you did in your screenshot.

Dark:
![image](https://user-images.githubusercontent.com/61414485/225465527-71a4365d-d08a-4abb-b18c-e67e2651f02c.png)
![image](https://user-images.githubusercontent.com/61414485/225465848-27a72710-31d9-4573-95bd-0fd50679c88f.png)


Light:
![image](https://user-images.githubusercontent.com/61414485/225465724-0ce6ccce-4ff6-4ee5-9e68-568fb4d87c55.png)
![image](https://user-images.githubusercontent.com/61414485/225465888-333265c9-3b30-486c-bb28-02d45cf4bec8.png)

Reply:
I think this is good!

I took my screenshot using Firefox’s “Simulate Achromatopsia” option in the Accessibility tab in dev tools, which is useful for testing this and other color vision variations:

<img width="438" alt="image" src="https://user-images.githubusercontent.com/357379/225609565-9f9c169b-4e40-421e-8352-addba7d8dcdd.png">